### PR TITLE
Makefile, update readme, rename title

### PR DESCRIPTION
**Pull request by jotak**
- Adding a makefile to have similar commands as in our other projects, including image building. Also there's a target to setup bridge.
- I changed the title from "Netflow traffic" to "Network Traffic" (more technology agnostic ; first words letter capitalized in titles, it's a common rule in openshift console)
- Switch to node 14, 12 being EOL soon
- Adding axios dependency for next tasks

### CODE REVIEW

**Comment by jotak** [comment]
(let me know if you'd suggest a different title for this page)

File: README.md [hunk 1]
```diff
@@ -2,51 +2,57 @@
...
-```
-$ npm build
-```
+This is equivalent to the npm commands `npm install && npm run build`.
 
-This should produce the following files :
+To run tests and linter:
 
-```
-$ tree dist/
```
Comment (jotak) [inline README.md:22]: I've also removed that, it doesn't seem super useful to me but let me know if you think otherwise :-) )

**Review by OlivierCazade — APPROVED** [review]
LGTM, about the title, I used netflow traffic because it was the one used in the mockups :
https://marvelapp.com/prototype/h4fei7h/screen/82371385

I also prefer network traffic.

**Comment by jotak** [comment]
@andrew-ronaldson I didn't realize the name was taken from the mockups; what do you think is better, between "Netflow traffic" and "Network Traffic" ? My rationale for changing is that "netflow" can be understood as a particular technology and maybe we want a more generic term (although it's true that we use IPFIX which is "kinda" netflow v10).

**Comment by andrew-ronaldson** [comment]
I like Network traffic @jotak . Seems to be more clear.

**Comment by jotak** [comment]
thanks @OlivierCazade @andrew-ronaldson

**Pull request merged**